tests/subsys/zbus/user_data/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+# SPDX-License-Identifier: Apache-2.0
+cmake_minimum_required(VERSION 3.13.1)
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+project(integration)
+
+list(APPEND HEADERS
+        "${CMAKE_CURRENT_SOURCE_DIR}/include"
+        )
+list(
+        APPEND SOURCES
+        "${CMAKE_CURRENT_SOURCE_DIR}/src/main.c"
+)
+
+include_directories(${HEADERS})
```

Comment:
How would you feel about moving those local headers file in the same directory as the code (`src`) and dropping the include directory from the samples? Keep things a bit simpler, no other sample use a dedicated local include and personally for small samples like this I find it easier to have everything together.